packages/atomic/src/components/recommendations/atomic-recs-interface/atomic-recs-interface.new.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import {parameters} from '@coveo/atomic-storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic-storybook-utils/common/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+
+async function initializeRecsInterface(canvasElement: HTMLElement) {
+  await customElements.whenDefined('atomic-recs-interface');
+  const recsInterface = canvasElement.querySelector('atomic-recs-interface');
+  await recsInterface!.initialize({
+    accessToken: 'xx149e3ec9-786f-4c6c-b64f-49a403b930de',
+    organizationId: 'fashioncoveodemocomgzh7iep8',
```

Comment:
We don't necessarily have to deal with this right now, but it bother me that we're showcasing a commerce use-case in storybook for this component.

Ideally, we'd show recommended articles or something of the sort, because we don't want people to associate the `atomic-recs-interface` (and by extension `atomic-recs-list`) to the commerce use case; they should think `atomic-commerce-recommendation-interface` / `atomic-commerce-recommendation-list` for that.

If you decide not to address this right away, I think we should create a tech debt Jira.

Reply:
https://coveord.atlassian.net/browse/KIT-3607